<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALU Simulator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/Resources/WindowIcon.png</normaloff>:/Resources/WindowIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="inputFrame">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>161</width>
      <height>394</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lnOper_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnOper_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblOp_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Operador 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblOp_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Operador 2:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="outputFrame">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>181</width>
      <height>394</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lnResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>192</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resultado: </string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="operatorFrame">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>11</y>
      <width>241</width>
      <height>394</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnSum">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMult">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDiv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAluImg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>200</width>
       <height>270</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Images.qrc">:/Resources/alusymbol.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblOperation">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblHexOp_1">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>140</y>
       <width>78</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblHexOp_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>248</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblHexRes">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>191</y>
       <width>74</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>btnSum</zorder>
    <zorder>btnMult</zorder>
    <zorder>btnDiv</zorder>
    <zorder>lblOperation</zorder>
    <zorder>lblHexOp_1</zorder>
    <zorder>lblHexOp_2</zorder>
    <zorder>lblHexRes</zorder>
    <zorder>lblAluImg</zorder>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
